content/es/docs/reference/glossary/kube-apiserver.md

```diff
@@ -0,0 +1,25 @@
+---
+title: API server
+id: kube-apiserver
+date: 2020-07-01
+full_link: /docs/reference/generated/kube-apiserver/
+short_description: >
+  Componente del plano de control que expone la API de Kubernetes.
+
+aka:
+- kube-apiserver
+tags:
+- architecture
+- fundamental
+---
+
+El API server es el componente del {{< glossary_tooltip text="plano de control " term_id="control-plane" >}}
+de Kubernetes que expone la API de Kubernetes. Se trata del frontend de Kubernetes,
+recibe las peticiones y actualiza acordemente el estado en {{< glossary_tooltip term_id="etcd" length="all" >}}.
+
+<!--more-->
+
+La principal implementación de un Kubernetes API server es
```

Comment:
I'd write “servidor” here even if leaving the title in English.

Reply:
Localised in https://github.com/kubernetes/website/pull/22259/commits/e423bf22516ddde02c86ee8d5e4e2f5c1c241930